appveyor.yml

```diff
@@ -1,15 +1,29 @@
-version: 1.0.{build}
+version: 0.{build}
+image: Visual Studio 2017
 configuration: Release
-assembly_info:
+
+environment:
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
+install:
+  - choco install gitversion -pre -y

```

Comment:
I assume that this line is installing (potentially) a pre-release version of gitversion. Is that what we want here? Surely we want the latest stable version.